src/firefly/js/ui/UploadTableChooser.js

```diff
@@ -268,28 +264,30 @@ export function showUploadTableChooser(setUploadInfo,groupKey= 'table-chooser',d
 
 const TapUploadPanel= ({setUploadInfo,groupKey= 'table-chooser',defaultColsEnabledObj}) => {
     return (
-    <FieldGroup groupKey={groupKey}>
-        <Stack style={{ resize: 'both', overflow: 'hidden', zIndex: 1, pt:1, minWidth: 600, minHeight: 500}}>
-            <FieldGroupTabs initialState={{value: 'upload'}} fieldKey='upload-type-tabs' groupKey={groupKey}
-                            sx={{width: 1,  flex: '1 1 auto'}}>
-                <Tab name='Upload Tables' id='upload' sx={{fontSize:'larger'}}>
-                    <FileUploadDropdown {...{
-                        sx:{height:1,
-                            '.ff-FileUploadViewPanel-file':{ml:3},
-                            '.ff-FileUploadViewPanel-acceptList':{ml:3},
-                        },
-                        acceptOneItem:true, acceptList:[TABLES], keepState:true, groupKey:groupKey+'-fileUpload',
-                        onCancel:() => dispatchHideDialog(dialogId),
-                        onSubmit:(request) => uploadSubmit(request,setUploadInfo,defaultColsEnabledObj),
-                    }}/>
-                </Tab>
-                <Tab name='Loaded Tables' id='tableLoad' sx={{fontSize:'larger'}}>
-                        <LoadedTables {...{
-                            style:{height: '100%', width:'100%'}, keepState: true, groupKey:groupKey+'-tableLoad',
-                            onCancel:() => dispatchHideDialog(dialogId),
-                            onSubmit:(request) => existingTableSubmit(request,setUploadInfo,defaultColsEnabledObj)
-                        }}/>
-                </Tab>
-            </FieldGroupTabs>
-        </Stack>
-    </FieldGroup> );};
+        <Box sx={{height:500, display: 'flex', flexDirection: 'column', resize: 'both', overflow: 'hidden', minHeight: 500, minWidth: 600}}>
+            <FieldGroup groupKey={groupKey} sx={{ flexGrow: 1 }}>
+                <Stack sx={{zIndex: 1, pt: 1, height: '100%'}}>
```

Comment:
don't need sx with stack

Reply:
Done. made the changes and re-added cancel buttons and made the PR against `rc-2024.1`.